components/Buttons/styleButtons.css

```diff
@@ -0,0 +1,81 @@
+@import url(../../css/avatar.css);
+@import url(../../css/color.css);
+@import url(../../css/padding.css);
+@import url(../../css/margin.css);
+@import url(../../css/border.css);
+@import url(../../css/font.css);
+@import url(../../css/background.css);
+@import url(../../css/button.css);
+
+html {
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  box-sizing: border-box;
+}
```

Comment:
repeated code, why body is used inside buttons? remove repeated code

Reply:
removed